assets/js/modules/analytics/dashboard/dashboard-widget-overview.js

```diff
@@ -46,21 +47,51 @@ import { __ } from '@wordpress/i18n';
 import { Component } from '@wordpress/element';
 
 class AnalyticsDashboardWidgetOverview extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			report: false,
+			directTotalUsers: false,
+		};
+	}
+	// When additional data is returned, componentDidUpdate will fire.
+	componentDidUpdate() {
+		this.processCallbackData();
+	}
+
+	componentDidMount() {
+		this.processCallbackData();
+	}
+
+	/**
+	 * Process callback data received from the API.
+	 */
+	processCallbackData() {
+		const {
+			data,
+			requestDataToState,
+		} = this.props;
+
+		if ( data && ! data.error && 'function' === typeof requestDataToState ) {
```

Comment:
I'm sure this is present in other components, but won't the component not be rendered by `withData` if there is no data, or if there is an error? It seems like those are unnecessary to check.

Reply:
Yea, likely this check can be removed. I can give that a try.

Reply:
removed these tests in https://github.com/google/site-kit-wp/pull/1277/commits/862cd5782ff1b3f655b5a315a367e3e9e21d1ce9 - everything worked fine after this change, with no console warnings.